src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -26,16 +26,24 @@ object WdlExpression {
 
   val unaryOperators = Set("LogicalNot", "UnaryPlus", "UnaryNegation")
 
-  def evaluate(ast: AstNode, lookup: ScopedLookupFunction, functions: WdlFunctions): Try[WdlValue] = {
+  private def replaceInterpolationTag(string: String, tag: String, lookup: ScopedLookupFunction) =
+    string.replace(tag, lookup(tag.substring(2, tag.length - 1)).valueString)
+
+  def interpolate(str: String, lookup: ScopedLookupFunction): String =
+    "\\$\\{([a-zA-Z]([a-zA-Z0-9_])*)\\}".r.findAllIn(str).foldLeft(str) { (string, tag) => replaceInterpolationTag(string, tag, lookup) }
```

Comment:
There's not really a point to those symbolic names though, they just take up space - because it's a fold you know one is the thing you're folding over and one is the accumulator.